clippy_lints/src/panic_unimplemented.rs

```diff
@@ -110,8 +110,8 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for PanicUnimplemented {
                               "`todo` should not be present in production code");
                 } else if is_expn_of(expr.span, "unreachable").is_some() {
                     let span = get_outer_span(expr);
-                    span_lint(cx, UNREACHABLE, span,
-                              "`unreachable` should not be present in production code");
+                    span_help_and_lint(cx, UNREACHABLE, span,
+                              "`unreachable` should not be present in production code", "ident is neither enable nor disable");
```

Comment:
That was not, what the issue was talking about. Anyway, the requested lint already exists, so I don't think there is anything that needs to be done here.

Reply:
Thank you for the review and sorry for the misunderstanding. I will fix this.